assets/js/components/notifications/ErrorNotifications.test.js

```diff
@@ -150,4 +154,70 @@ describe( 'ErrorNotifications', () => {
 		);
 		expect( container ).toMatchSnapshot();
 	} );
+
+	it( 'does render the redo setup CTA if initial Site Kit setup auth is not granted', () => {
```

Comment:
Let's avoid the abbreviation here too, for the sake of clarity:

```suggestion
	it( 'does render the redo setup CTA if initial Site Kit setup authentication is not granted', () => {
```